prow/github/client.go

```diff
@@ -1426,3 +1426,22 @@ func (c *Client) IsMergeable(org, repo string, number int, sha string) (bool, er
 	}
 	return false, fmt.Errorf("reached maximum number of retries (%d) checking mergeability", maxTries)
 }
+
+// ClearMilestone clears the milestone from the specified issue
+func (c *Client) ClearMilestone(org, repo string, num int) error {
+	c.log("ClearMilestone", org, repo, num)
+	if c.fake {
```

Comment:
I don't think you need this check. If `c.fake` is true `c.request()` will react appropriately.

Reply:
Done.